### PR TITLE
Teshari sprite pack 5: Blood Money

### PR DESCRIPTION
**Pull request by Ebin-Halcyon**
## About The Pull Request

Updates a few bunch of GAGS clothing and some fallback gear, but mostly updates Cargo and the Syndicate clothing.

Teshari PR

image
image
image

## How This Contributes To The Skyrat Roleplay Experience

Fixes and cleanup to a species that everyone else refuses to sprite for.

## Changelog

:cl:
imageadd: Fixes a few GAGs'd items to have support for Teshari, namely sundresses and plaid skirts, updates the scarf and glove fallback sprites, check them out.
imageadd: Syndicate and cargo gear has been resprited to fit on Teshari
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
Requesting DNM until #15257 - Spritefile cleanup for several related items, specifically GAGS skirts
That one has changes to the greyscale configs that I'd rather you dont have to deal with conflicts of

**Comment by Ebin-Halcyon** [comment]
> Requesting DNM until #15257 - Spritefile cleanup for several related items, specifically GAGS skirts That one has changes to the greyscale configs that I'd rather you dont have to deal with conflicts of

All good with me, I got more than enough time to wait. 👍

File: code/modules/clothing/under/skirt_dress.dm [hunk 1]
```diff
@@ -49,6 +49,7 @@
 	greyscale_config = /datum/greyscale_config/plaidskirt
 	greyscale_config_worn = /datum/greyscale_config/plaidskirt_worn
 	greyscale_config_worn_digi = /datum/greyscale_config/plaidskirt_worn/digi //SKYRAT EDIT - DigiGreyscale
+	greyscale_config_worn_teshari = /datum/greyscale_config/plaidskirt_worn/teshari //SKYRAT EDIT - Teshari greyscale
```
Comment (GoldenAlpharex) [inline code/modules/clothing/under/skirt_dress.dm]: I feel like we could just place those overrides in a modular file instead, no?

File: code/modules/clothing/under/skirt_dress.dm [hunk 2]
```diff
@@ -79,4 +80,5 @@
 	greyscale_colors = "#FFE60F#9194A5#1F243C"
 	greyscale_config = /datum/greyscale_config/sundress
 	greyscale_config_worn = /datum/greyscale_config/sundress_worn
+	greyscale_config_worn_teshari = /datum/greyscale_config/sundress_worn/teshari
```
Comment (GoldenAlpharex) [inline code/modules/clothing/under/skirt_dress.dm]: Ditto (plus you forgot the modular comment here)

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]

**Comment by Ebin-Halcyon** [comment]
Same as the bluesec PR, gonna add more to this when i'm more motivated, closing it so it doesn't go stale.

**Comment by Ebin-Halcyon** [comment]
Alright let's get it, back to the slave life.

**Review by tf-4 — APPROVED** [review]

File: modular_skyrat/modules/GAGS/greyscale_configs.dm [hunk 3]
```diff
@@ -785,6 +785,16 @@
...
 	icon_file = 'modular_skyrat/master_files/icons/mob/clothing/under/skirts_dresses.dmi'
 
+/datum/greyscale_config/plaidskirt_worn/teshari
+	name = "Worn Plaid Skirt - Teshari"
+	icon_file = 'modular_skyrat/modules/GAGS/icons/dressteshari.dmi'
+
+/datum/greyscale_config/sundress_worn/teshari
+	name = "Worn Sundress - Teshari"
+	icon_file = 'modular_skyrat/modules/GAGS/icons/dressteshari.dmi'
+	json_config = 'code/datums/greyscale/json_configs/sundress_worn.json'
```
Comment (Tastyfish) [inline modular_skyrat/modules/GAGS/greyscale_configs.dm]: Line unnecessary as that comes with being a subtype of `/datum/greyscale_config/sundress_worn`
```suggestion
```

**Review by Tastyfish — CHANGES_REQUESTED** [review]

**Review by Tastyfish — APPROVED** [review]
code

**Review by GoldenAlpharex — APPROVED** [review]
Looks good!

**Pull request merged**